disperser/dataapi/server.go

```diff
@@ -402,6 +418,49 @@ func (s *server) FetchOperatorsNonsigningPercentageHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, metric)
 }
 
+// FetchDeregisteredOperators godoc
+//
+//	@Summary	Fetch list of DeregisteredOperators for days
+//	@Tags		OperatorsInfo
+//	@Produce	json
+//	@Success	200		{object}	BlobsResponse
+//	@Failure	400		{object}	ErrorResponse	"error: Bad request"
+//	@Failure	404		{object}	ErrorResponse	"error: Not found"
+//	@Failure	500		{object}	ErrorResponse	"error: Server error"
+//	@Router		/operatorsInfo/deRegisteredState [get]
```

Comment:
`deRegisteredState` --> `deRegisteredOperators`

also add an example query param for completeness sake?